src/cgame/rocket/rocket.cpp

```diff
@@ -386,18 +386,13 @@ void Rocket_Init()
 
 void Rocket_Shutdown()
 {
+	// TODO: Consider that it may be more efficient to do nothing at all here, in
```

Comment:
The cgame gets restarted several times and in so lib mode, we will leak resources.